src/components/input/index.js

```diff
@@ -0,0 +1,32 @@
+import { h } from 'preact';
```

Comment:
It looks to me that this select component is too specific to the needs of pirania ui, and so it shouldn't be included as an app-wise component, as it won't be easily reused.